### ISSUE TITLE
patientproblemcodes.csv renamed?

### ISSUE THREAD

**Issue by jadczak-sibel** [jadczak/mauder#1]
It looks like patient problem codes was renamed in the latest archives to "patientproblemcode.csv"  singular.

```
Traceback (most recent call last):
  File "C:\dev\mauder\mauder.py", line 643, in <module>
    main(argv[1:])
  File "C:\dev\mauder\mauder.py", line 49, in main
    patient_codes = parse_patient_codes(patient_codes_file)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\dev\mauder\mauder.py", line 453, in parse_patient_codes
    with open(patient_codes_file, "rb") as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\dev\\mauder\\mdr-data-files\\patientproblemdata\\patientproblemcodes.csv'
```

**Comment by jadczak-sibel** [comment]
Dumber than that.  The format of the file was changed as well.
The old file format was `CODE,PROBLEM` now it's `CODE,PROBLEM,NCIT_CODE,IMDRF_CODE`

Which is probably causing problems since there is now a comma in the parsed text...

![image](https://github.com/user-attachments/assets/f9feecc6-4639-4d43-8317-993cdbabe6d3)

**Comment by jadczak-sibel** [comment]
Also there seem to be commas in some of the problem codes.  Was that a thing before?

**Comment by jadczak** [comment]
Fixed in e09d33077043fef048edabcea2896c281a1ecb14
